reference/configuration/framework.rst

```diff
@@ -465,45 +953,211 @@ would be ``/images/logo.png?version=5``.
 
-.. versionadded:: 2.2
-    The ``enabled`` option was introduced in Symfony 2.2. Previously, the profiler
-    could only be disabled by omitting the ``framework.profiler`` configuration
-    entirely.
+**type**: ``string`` **default**: ``null``
 
-.. _profiler.enabled:
+Sets the content shown during the loading of the fragment or when JavaScript is
+disabled. This can be either a template name or the content itself. It becomes
+the service container parameter named ``fragment.renderer.hinclude.global_template``.
 
-enabled
+.. seealso::
+
+    See :ref:`book-templating-hinclude` for more information about hinclude.
+
+.. _reference-templating-form:
+
+form
+....
+
+resources
+"""""""""
+
+**type**: ``string[]`` **default**: ``['FrameworkBundle:Form']``
+
+A list of all resources for form theming in PHP. If you have custom global form
+themes in ``src/WebsiteBundle/Resources/views/Form``, you can configure this like:
+
+.. configuration-block::
+
+    .. code-block:: yaml
+
+        # app/config/config.yml
+        framework:
+            templating:
+                form:
+                    resources:
+                        - 'WebsiteBundle:Form'
+
+    .. code-block:: xml
+
+        <!-- app/config/config.xml -->
+        <?xml version="1.0" encoding="UTF-8" ?>
+        <container xmlns="http://symfony.com/schema/dic/services"
+            xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+            xmlns:framework="http://symfony.com/schema/dic/symfony"
+            xsi:schemaLocation="http://symfony.com/schema/dic/services http://symfony.com/schema/dic/services/services-1.0.xsd
+                http://symfony.com/schema/dic/symfony http://symfony.com/schema/dic/symfony/symfony-1.0.xsd">
+
+            <framework:config>
+
+                <framework:templating>
+
+                    <framework:form>
+
+                        <framework:resource>WebsiteBundle:Form</framework:resource>
+
+                    </framework:form>
+
+                </framework:templating>
+
+            </framework:config>
+        </container>
+
+    .. code-block:: php
+
+        // app/config/config.php
+        $container->loadFromExtension('framework', array(
+            'templating' => array(
+                'form' => array(
+                    'resources' => array(
+                        'WebsiteBundle:Form'
+                    ),
+                ),
+            ),
+        ));
+
+.. note::
+
+    The default form templates from ``FrameworkBundle:Form`` will always be
+    included in the form resources.
+
+.. _reference-templating-base-urls:
+
+assets_base_urls
+................
+
+**type**: ``{ http: [], ssl: [] }``
+
+This option allows you to define base URLs to be used for assets referenced
+from ``http`` and ``ssl`` (``https``) pages. A string value may be provided in
+lieu of a single-element array. If multiple base URLs are provided, Symfony2
+will select one from the collection each time it generates an asset's path.
```

Comment:
The rules of possessive words:

`-'s` the general rule
`-'` when it's a plural word that ends with `s`
`[..] of [..]` for 2 objects

E.g.: "Symfony's Documentation" and "Johannes's Serializer lib", but "the boys' profiles" and "the roof of the house".